scripts/host-aware-build.sh

```
@@ -24,10 +24,8 @@ shift || true
 os_name=$(uname -s || echo unknown)
```

Shellcheck (0.11.0)

[warning] os_name appears unused. Verify use (or export if used externally).

(SC2034)